app/routes/projects/$projectId/index.tsx

```diff
@@ -365,9 +369,15 @@ export default function ProjectDetailsPage() {
                 <Card>
                   <CardHeader title="Slack Channel:" />
                   <CardContent>
-                    <Stack direction="row" spacing={1}>
+                    <Link
+                      target="_blank"
+                      href={`https://wizeline.slack.com/channels/${project.slackChannel.replace(
```

Comment:
Why is this replace need it? should we take care of other # in the string or just one?

Reply:
we should add a new validation in the project form, to ask a # in that field

Reply:
I'm only removing `#` if found. Some people write it, some don't. This should work for both cases. I agree that we could add a validation to either make sure it has it, or it doesn't 😛